hass-game-db-updater/config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Playnite Web Game DB Updater",
+    "url": "https://github.com/andrew-codes/playnite-web",
+    "version": "3-dev",
```

Comment:
Maybe this is a good time to introduceIf one doesn't already exist, this might be a good opportunity to introduce a common location for the version to be read from. This way, it can be reused across the CI pipeline.